include/stdexec/__detail/__basic_sender.hpp

```diff
@@ -39,14 +39,10 @@ namespace stdexec {
     using __impl_of = decltype((__declval<_Sender>().__impl_));
   } // namespace __detail
 
-  template <
-    class _Descriptor,
-    auto _DescriptorFn =
-      [] {
-        return _Descriptor();
-      }
-  >
-  inline constexpr auto __descriptor_fn_v = _DescriptorFn;
+  template <class _Descriptor>
+  inline constexpr auto __descriptor_fn_v = [] {
+    return _Descriptor();
+  };
```

Comment:
see [this comment](https://github.com/NVIDIA/stdexec/pull/1405#issuecomment-2565888010) for why i can't accept this change.